src/main/java/org/apache/pulsar/io/sftp/sink/FileWriteThread.java

```diff
@@ -32,6 +31,10 @@
  * and publishes them to a Pulsar topic.
  */
 public class FileWriteThread extends Thread {
+    public static final String FILE_NAME = "file.name";
```

Comment:
Can use constants defined by org.apache.pulsar.io.sftp.utils.Constants

Reply:
yes, this needs to be moved out later, I will modify it.